wxCompile.nim

```diff
@@ -3,16 +3,23 @@
 
 include ospaths
 
+when defined(wxWidgetsPath):
+  const wxWidgetsPath {.strdefine.}: string = nil
+  when len(wxWidgetsPath) < 1:
+    {.error: "wxWidgetsPath must not be an empty string when defined".}
+else:
+  when defined(windows):
+    {.error: "wxWidgetsPath must be defined to compile with wxWidgets".}
```

Comment:
This should not be an error! Even on Windows, you could setup your build-environment so that the wxWidgets artifacts are _somehow_ known to the compiler when it is invoked. There is **NO** fundamental difference to linux here!